src/components/formControls/input/style.ts

```diff
@@ -3,32 +3,50 @@
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import styled from 'styled-components'
+import theme from '../../../theme/brave-default'
+import { Props } from './index'
 
-export const StyledInput = styled.input`
+export const InputComponent = styled.div`
+  background-color: #fff;
   min-height: auto;
   box-sizing: border-box;
   width: 100%;
-  max-width: 100%;
-  font-size: 14px;
-  font-family: "Poppins", sans-serif;
+  max-width: 100;
   border: 1px solid #DFDFE8;
   border-radius: 3px;
-  padding: 12px 10px;
-  color: #686978;
-  outline: unset;
+  padding: 8px 10px;
+  opacity: ${(p: Props) => p.disabled ? '0.3' : '1'};
 
-  &:focus {
+  &:focus-within {
     border-color: #A1A8F2;
   }
+` as any
```

Comment:
Can you please remove any?

More info here https://github.com/brave/brave-ui/wiki/Code-style-guide#styled-component-definition